Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.40995&attempt_id=c0b2d690-680a-4da4-ba57-206a10b3e2b3)

Attempted: python2Packages.fpylll, python3Packages.fpylll

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/6p0715xklf0qr64ia3jqaav47j31ds2j-python2.7-numpy-1.14.2/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it by " \
  ^~~~~~~
building of '/nix/store/xpg1zk3gwmlapnsvlmvw3p3zizs7c3ky-python2.7-Cython-0.28.1.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/i9ixr29088pfvf859bg98b21p173dig5-python2.7-cysignals-1.7.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/4h4hpd4n82x7m7jjgamgyljvh8rdjz5m-python2.7-fpylll-0.4.0dev.drv': 2 dependencies couldn't be built
building of '/nix/store/xpci0kq0rbsvj7ag9an298cy2z34skhy-python3.6-Cython-0.28.1.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/qp0q7gwpwbqmblldjki2y8xh2qd4f48n-python3.6-cysignals-1.7.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7vb33x30xj0jkkqpbv3gac4hfcjfqfky-python3.6-fpylll-0.4.0dev.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/4h4hpd4n82x7m7jjgamgyljvh8rdjz5m-python2.7-fpylll-0.4.0dev.drv', '/nix/store/7vb33x30xj0jkkqpbv3gac4hfcjfqfky-python3.6-fpylll-0.4.0dev.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/fplll/default.nix
+++ b/pkgs/development/libraries/fplll/default.nix
@@ -4,12 +4,12 @@
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "fplll";
-  version = "5.2.0";
+  version = "5.2.1";
   src = fetchFromGitHub {
     owner = "${pname}";
     repo = "${pname}";
     rev = "${version}";
-    sha256 = "0931i4q49lzlifsg9zd8a2yzj626i1s2bqhkfxvcxv94c38s0nh1";
+    sha256 = "015qmrd7nfaysbv1hbwiprz9g6hnww1y1z1xw8f43ysb7k1b5nbg";
   };
   nativeBuildInputs = [autoconf automake libtool gettext autoreconfHook];
   buildInputs = [gmp mpfr];

--- a/pkgs/development/python-modules/fpylll/default.nix
+++ b/pkgs/development/python-modules/fpylll/default.nix
@@ -1,5 +1,5 @@
 { lib
-, fetchPypi
+, fetchFromGitHub
 , buildPythonPackage
 , pkgconfig
 , gmp
@@ -14,11 +14,13 @@
 
 buildPythonPackage rec {
   pname = "fpylll";
-  version = "0.3.0dev";
+  version = "0.4.1dev";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0bjkh02fnxsrxwjzai8ij12zl2wq319z8y25sn9pvvzla5izgnp9";
+  src = fetchFromGitHub {
+    owner = "fplll";
+    repo = "fpylll";
+    rev = version;
+    sha256 = "01x2sqdv0sbjj4g4waj0hj4rcn4bq7h17442xaqwbznym9azmn9w";
   };
 
   buildInputs = [

